main.rb

```diff
@@ -0,0 +1,101 @@
+require './app'
+
+@app = App.new
+
+OPTIONS = {
+  1 => :list_all_books,
+  2 => :list_all_people,
+  3 => :create_person,
```

Comment:
**You did great job!**

However, we have some places to fix up. Let me point them out :)

- [ ] For starting, in creating person there is an issue which is, it's not matching with the original design. In original design user can select person type by 1 or 2 but in your project you need to write student or teacher to create. This approach is going to take user's time. Please stick to the original design:

<img width="421" alt="Screenshot 2023-06-22 at 22 18 03" src="https://github.com/danielamoreno699/school-library/assets/69011963/26e808b1-f06f-4c5f-8a9a-37dd729d2898">